packages/frontend/core/src/modules/journal/store/journal.ts

```diff
@@ -37,6 +37,18 @@ export class JournalStore extends Store {
     });
   }
 
+  getDocJournalDate(docId: string) {
```

Comment:
I think we are better using `watch().value`, which is slower but usually not a performance bottleneck.

I think get() is prone to abuse, and the data is not real-time, which leads to inconsistency.

the codes for `get` and `watch` are usually not shared, which causes us to maintain two sets of data reading logic.